src/permissions.c

```diff
@@ -26,6 +26,55 @@
 
 static XdpDbusImplPermissionStore *permission_store = NULL;
 
+static gboolean
```

Comment:
Why not return GStrv and get rid of `out_permissions`?

Reply:
Well I liked more to check the return value and based on that react on caller function, but both are fine options.